manga_translator/translators/groq.py

```
@@ -0,0 +1,155 @@
+import groq
+import asyncio
```

Ruff

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)